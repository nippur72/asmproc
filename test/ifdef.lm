TEXT
#ifdef DEBUG then DEBUG_IS_ON
TEXT
#ifndef DEBUG then DEBUG_IS_OFF
TEXT
