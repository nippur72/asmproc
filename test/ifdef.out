TEXT
#IFDEF DEBUG
   DEBUG_IS_ON
#ENDIF
TEXT
#IFNDEF DEBUG
   DEBUG_IS_OFF
#ENDIF
TEXT
